recipes/python/build.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+
+import multiprocessing
+import os
+import re
+import subprocess
+import sys
+
+SOURCE_TAR = "src/Python-{version}.tgz"
+
+CONFIGURE_ENV = {
+    "linux_x86_64_rocky-8.5": {
+        "CFLAGS": "-O2 -g -pipe -Wall -Wp,-D_FORTIFY_SOURCE=2 -fexceptions "
+                  "-fstack-protector-strong --param=ssp-buffer-size=4 "
+                  "-grecord-gcc-switches -m64 -mtune=generic -D_GNU_SOURCE "
+                  "-fPIC -fwrapv",
+        "LDFLAGS": "-Wl,-z,relro,-rpath={config.package_path}/lib"
+    }
+}
+
+CONFIGURE_OPTS = {
+    "linux_x86_64_rocky-8.5": #"--build=x86_64-unknown-linux-gnu "
+                              #"--host-x86_64-unknown-linux-gnu "
+                              "--prefix={config.package_path} --enable-ipv6 "
+                              "--enable-shared "
+                              "--with-dbmliborder=gdbm:ndbm:bdb "
+                              "--with-system-expat --with-system-ffi "
+                              # "--enable-optimizations "
+                              "--without-ensurepip "
+}
+
+def build():
+    build_config = BuildConfig()
```

Comment:
for sake of decreasing verbosity, consider making this a global, then not having to pass it around to various funcs

Reply:
done